src/Integration.UnitTests/Binding/BindCommandTests.cs

```diff
@@ -220,10 +223,11 @@ public void BindCommand_SetBindingInProgress_ProjectViewModelIsBoundChanges()
 
             BindCommand testSubject = this.PrepareCommpandForExecution(new ConnectionInformation(serverVM.Url));
             var progressEvents = new ConfigurableProgressEvents();
-            testSubject.Controller.ConnectedServers.Add(serverVM);
-            testSubject.Controller.BoundProjects.Add(projects[0]);
+            testSubject.Controller.State.ConnectedServers.Add(serverVM);
+            testSubject.Controller.State.SetBoundProject(projects[0]);
```

Comment:
I've moved some of the code in the controller into the state object - specifically logic that deals with bound projects (more about that below).